akka-docs-dev/rst/scala/code/docs/http/scaladsl/server/directives/PathDirectivesExamplesSpec.scala

```diff
@@ -16,7 +16,7 @@ class PathDirectivesExamplesSpec extends RoutingSpec {
 
   //# path-dsl
   // matches /foo/
-  path("foo" /)
+  path("foo"./)
```

Comment:
Unfortunately the linter is not smart enough to recognize that, which was the reason for abandoning postfix notation for durations as well.